### PR TITLE
Tests cloud_networks with RBAC filtering from super

### PR DESCRIPTION
**Pull request by d-m-u**
Amazon's allowed_cloud_networks needs test for RBAC filtering from PR [here](https://github.com/ManageIQ/manageiq/pull/16824/files)

### CODE REVIEW

**Comment by d-m-u** [comment]
@miq-bot add_label test

**Comment by d-m-u** [comment]
@syncrou can you review please?

**Comment by d-m-u** [comment]
@Ladas does this work for https://github.com/ManageIQ/manageiq/pull/16824#discussion_r161679886?

**Comment by bronaghs** [comment]
@d-m-u Once you rebase, the tests should pass.

File: spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb [hunk 1]
```diff
@@ -2,6 +2,11 @@
   include Spec::Support::WorkflowHelper
 
   let(:admin) { FactoryGirl.create(:user_with_group) }
+  let(:provider) do
+    allow(User).to receive_messages(:server_timezone => "UTC")
+    FactoryGirl.create(:ems_amazon)
+  end
+
```
Comment (bronaghs) [inline spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb]: @d-m-u - can you use` :ems ` instead of creating `:provider`?

**Comment by bronaghs** [comment]
@gmcculloug - bump

**Comment by d-m-u** [comment]
@miq-bot add_label gaprindashvili/yes

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 850cb12ec628e2ed1e5f38f548b19efc582136a1
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Fri Mar 9 06:39:50 2018 -0500

    Merge pull request #394 from d-m-u/adding_test_for_allowed_ci
    
    Tests cloud_networks with RBAC filtering from super
    (cherry picked from commit 6f051ff77df34c4eaed236c837132dac1c694c84)
```

**Pull request merged**